src/views/record-browser/query-builder/query-builder.scss

```diff
@@ -10,27 +10,20 @@
 		h4 {
 			color: $secondary;
 		}
-
-		button {
-			width: 100%;
-			margin-left: 0;
-			margin-bottom: $padding;
-		}
 	}
 
-	.add-filter-button {
+	.add-filter-button.md-button {
 		background: $blue-gray;
-		height: 22px;
-		width: 22px;
+		min-width: 22px;
+		min-height: 22px;
```

Comment:
We already talked about being hacky. 

We need to guarantee that this button stays a circle. If the contents exceed a height or width of 22px, then the circle becomes an oval.

If you want to continue to use `md-button`, then reset `min-width` and `min-height` to their default values.
```
min-width: initial;
min-height: initial;
height: 22px;
width: 22px;
```